frontend/app/routes/index.js

```diff
@@ -15,7 +15,8 @@ export default Ember.Route.extend({
     let startOfWeek = +moment().subtract(1, 'week').startOf('week')
     let assignments = this.store.find('assignment', { user, from: startOfWeek })
     let attendances = this.store.find('attendance', { user, from: +moment(day, 'YYYY-MM-DD') })
+    let activities  = this.store.find('activity')
```

Comment:
This could kill the whole app, this will fetch _all_ activities ever created, should probably limit current user and to last month like attendances